NetCoreRestApi/DataLayer/UnitOfWorks/Interfaces/ICategoryUnitOfWork.cs

```diff
@@ -0,0 +1,8 @@
+﻿using DataLayer.Entities;
+
+namespace DataLayer.UnitOfWorks.Interfaces
+{
+    public interface ICategoryUnitOfWork<TId> : IUnitOfWork<CategoryEntity<TId>, TId>
```

Comment:
Fix to ICategoryUnitOfWork : IUnitOfWork<CategoryModel, TId>

Reply:
done